MetaLamp/nft-marketplace/src/Plutus/Types/Percentage.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE DeriveAnyClass        #-}
+{-# LANGUAGE DeriveGeneric         #-}
+{-# LANGUAGE DerivingStrategies    #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TemplateHaskell       #-}
+
+module Plutus.Types.Percentage where
```

Comment:
I think this module should belong to `MetaLamp/nft-marketplace/src/Plutus/Abstract` in current architecture

Reply:
done